schemas/records.json

```diff
@@ -70,7 +72,10 @@
             }
           }
         }
-      ]
+      ],
+      "options": {
+        "hidden": true
+      }
```

Comment:
So that's set to null by default?

Reply:
Yes, currently it's set to null by the enum, and including it in the GUI actually offers no way to choose anything else. So I hid it completely. Do we need time for workflows and experiments? Again, trying to have as little properties as possible.

Reply:
Okay, then it should be fine as it is for now.